redhat-cop/uncontained.io
# Adding updated logo files; removing gulp-generated artifacts

#### What is this PR About?

- Adding new, slimmer logo images
- adjusted header spacing to account for slimmer image
- removed the gulp-generated stylesheets and added them to .gitignore

#### How should we test or review this PR?

Build the site, ensure header/logo spacing still looks good.
```
cd uncontained.io/themes/uncontained.io
npm run build
cd ../..
hugo serve -D --noHTTPCache
```
#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
